changelogs/fragments/6513-opkg-default-force.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

I think this should be `minor_changes`, as it is not really a bugfix. (There is no bug, it's just strange...)
```suggestion
minor_changes:
```